docs/modules/faqs.html

```diff
@@ -251,10 +274,18 @@ <h3>When loading a webpage in iOS WebView, my application gets a TypeError: unde
 						<h3>I notice that my clients lose audio and drop from the meeting if the Chrome browser on Android stays in background for several minutes, is that a known issue?</h3>
 					</a>
 					<p>This is a known issue and happens primarily due to this <a href="https://bugs.chromium.org/p/chromium/issues/detail?id=951418">Chromium bug</a>. At present we do not have a workaround.</p>
+					<a href="#i-only-see-iphone-or-ipad-microphone-as-audio-input-in-ios-device-is-that-a-known-issue" id="i-only-see-iphone-or-ipad-microphone-as-audio-input-in-ios-device-is-that-a-known-issue" style="color: inherit; text-decoration: none;">
+						<h3>I only see iPhone or iPad microphone as audio input in iOS device, is that a known issue?</h3>
```

Comment:
what is the issue here? can we elaborate ont he problem?

Reply:
I only see iPhone or iPad microphone as an audio input in my iOS device, is that a known issue?